card/lib/generators/card/format/USAGE

```diff
@@ -0,0 +1,9 @@
+Description:
```

Comment:
did you delete and add in separate commits?  looks like git didn't pick this up as a rename.